src/components/Property.astro

```diff
@@ -8,32 +8,15 @@ const formattedType = type.toLowerCase();
 const link = customDataTypes.includes(formattedType) ? '/api/data-types#' + formattedType :undefined;
 ---
 
-<li class="m-0 px-0 py-4 first:pt-0 last:pb-0">
-  <dl class="m-0 flex flex-wrap items-center gap-x-3 gap-y-2">
-    <dt class="sr-only">
-      Name
-    </dt>
-    <dd>
-      {name}
-    </dd>
-    <dt class="sr-only">Type</dt>
-    <dd>
-      <Badge text={formattedType} link={link}>
-    </dd>
-    { required &&
-      <dt class="sr-only">Required</dt>
-      <dd> <Badge type="required"> </dd> }
-    { experimental &&
-      <dt class="sr-only">Experimental</dt>
-      <dd> <Badge type="experimental"> </dd> }
-    { deprecated &&
-      <dt class="sr-only">Deprecated</dt>
-      <dd> <Badge type="deprecated"> </dd> }
-    <dt class="sr-only">
-      Description
-    </dt>
-    <dd class="w-full flex-none [&>:first-child]:mt-0 [&>:last-child]:mb-0">
+<li class="m-0 px-0 py-4 first:pt-0 last:pb-0 space-y-2">
+  <div class="m-0 flex flex-wrap items-center gap-x-3">
+    <h4 class="my-0" id={name}>{name}</h4>
+      <Badge text={formattedType} link={link} />
```

Comment:
Suggestion: It looks like the nesting here is incorrect i.e. these classes are tabbed in one too many.
We need to add better linting for Astro files to fix this.